templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -179,43 +179,46 @@ icon. The second positional argument defines a list of categories for the
 integration.
 
 At this point, if you're following along and/or writing your own Hello World
-webhook, you have written enough code to test your integration.
+webhook, you have written enough code to test your integration. There are three
+tools which you can use to test your webhook - 2 command line tools and a GUI.
 
-First, get an API key from the Your bots section of your Zulip user's Settings
-page. If you haven't created a bot already, you can do that there. Then copy
-its API key and replace the placeholder `<api_key>` in the examples with
-your real key. This is how Zulip knows the request is from an authorized user.
+## Step 4: Manually testing the webhook
 
-Now you can test using Zulip itself, or curl on the command line.
+For either one of the command line tools, first, you'll need to get an API key
+from the **Your bots** section of your Zulip user's Settings page. If you
+haven't created a bot already, you can do that there. If you plan on using the
+GUI, make sure that the bot is of the "**Incoming Webhook**" type Then copy its API
+key and replace the placeholder `<api_key>` in the examples with your real key.
+This is how Zulip knows the request is from an authorized user.
```

Comment:
This is way too complicated. Just make this:

```
To test the webhook, you'll need to [create a bot](https://zulipchat.com/help/add-a-bot-or-integration) 
with the **Incoming Webhook** type. Replace `<api_key>` with your bot's API key in the examples
presented below! This is how Zulip knows that the request was made by an authorized user.
```

Reply:
Ok, I'm starting to see how this is too complicated. The "if statement" added a bit too much unnecessary complexity. For the time being, I'll use the exact lines you've suggested here. :)